types/react-router/index.d.ts

```diff
@@ -61,7 +61,7 @@ declare module 'react-router' {
 
   interface RouteProps {
     location?: H.Location;
-    component?: React.SFC<RouteComponentProps<any> | void> | React.ComponentClass<RouteComponentProps<any> | void>;
+    component?: React.SFC<RouteComponentProps<any> | void | {}> | React.ComponentClass<RouteComponentProps<any> | void | {}>;
```

Comment:
I try tiny example for this
`const foo : RouteComponentProps<any> = {}`

but that cause error

```
 Type '{}' is not assignable to type 'RouteComponentProps<any>'.
  Property 'match' is missing in type '{}'.
```

If `RouteComponentProps<any>` include every value but null, 
its may be like below.

```ts
  interface RouteComponentProps<P> {
    match?: match<P>;
    location?: H.Location;
    history?: H.History;
  }
```

Should I change `RouteComponentProps`?